### PR TITLE
Added test cases for Geometry Defects MAGN-7970 and 8039

### PR DESCRIPTION
**Pull request by riteshchandawar**
### Purpose

Adding automation for Geometry defects.
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7970
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8039

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files

### Reviewers

@aparajit-pratap

### CODE REVIEW

File: test/Libraries/WorkflowTests/GeometryDefectTests.cs [hunk 1]
```diff
@@ -504,6 +504,59 @@ public void TestCoordinateSystem()
...
+            // check all the nodes and connectors are loaded
+            Assert.AreEqual(9, CurrentDynamoModel.CurrentWorkspace.Nodes.Count());
+            Assert.AreEqual(11, CurrentDynamoModel.CurrentWorkspace.Connectors.Count());
+
+            var VoronoiNodeID = "2fdd3ad8-dc1e-4087-be18-196732d41d7a";
+
+            for (int i = 0; i <= 10; i++)
+            {
+                var allLines = GetPreviewValueAtIndex(VoronoiNodeID, i) as Line;
+                Assert.IsNotNull(allLines);
```
Comment (aparajit-pratap) [inline test/Libraries/WorkflowTests/GeometryDefectTests.cs]: @riteshchandawar can you check for specific values returned from this node? In this case, even if the node returns incorrectly, the test may still pass.

Comment (riteshchandawar) [inline test/Libraries/WorkflowTests/GeometryDefectTests.cs]: Actually I kept it random and not checking for the total number of lines as well. I doubt I can do the verification based on any particular line. The main issue was it was throwing unhandled exception, which will definitely caught by this test case.

Comment (aparajit-pratap) [inline test/Libraries/WorkflowTests/GeometryDefectTests.cs]: Can you remove the random node?

**Comment by ikeough** [comment]
@riteshchandawar @aparajit-pratap This is pretty old guys. Can we fix it and bring it in?

**Pull request merged**